Week2/homework/maartjes-work.js

```diff
@@ -45,17 +45,33 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
+function computeTasksInHours(maartjesTasksX, celling) {
+  const marrtjeTasksInHours = maartjesTasksX
+    .map(task => task.duration / 60)
+    .filter(task => task >= celling);
```

Comment:
The result of mapping tasks to hours is an array of hours. Therefore you are filtering `hours` here and a single element of an `hours` array is an `hour`:

```js
.filter(hour => hour >= threshold);
```